.dockerignore

```diff
@@ -15,8 +15,8 @@
 **/kodata/
 **/*_test.go
 **/c.out
-**/*-coverage.html
-**/*-unit-tests.xml
```

Comment:
From what I see the pattern still contains a dash?

Reply:
fixed